File: Examples/VisualBasic/AsposePdfFacades/Attachments/AddAttachment.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Pdf
Imports Aspose.Pdf.Facades

Namespace AsposePDFFacades.Attachments
    Public Class AddAttachment
        Public Shared Sub Run()
            ' ExStart:AddAttachment
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdfFacades_Attachments()
            ' Open document
            Dim contentEditor As New PdfContentEditor()
            contentEditor.BindPdf(dataDir & "AddAttachment.pdf")

            ' Add attachment
            contentEditor.AddDocumentAttachment(dataDir & "test.txt", "Attachment Description")

            ' Save updated PDF
            contentEditor.Save(dataDir & "AddAttachment_out_.pdf")
            ' ExEnd:AddAttachment
        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePdfFacades/Attachments/AddAttachment.cs

using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.Facades;

namespace Aspose.Pdf.Examples.CSharp.AsposePDFFacades.Attachments
{
    public class AddAttachment
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Attachments();
            //open document
            PdfContentEditor contentEditor = new PdfContentEditor();
            contentEditor.BindPdf(dataDir+ "AddAttachment.pdf");

            //add attachment
            contentEditor.AddDocumentAttachment(dataDir+ "test.txt", "Attachment Description");

            //save updated PDF
            contentEditor.Save(dataDir+ "AddAttachment_out.pdf");
            
        }
    }
}